docs/examples/hairpin_fitting/hairpin_unfolding.rst

```diff
@@ -0,0 +1,346 @@
+
+    fit["dna_ss_hairpin/Lc"].value = 0.02  # in microns
+    fit["dna_ss_hairpin/Lc"].lower_bound = 0.001  
+
+    fit["dna_ss_hairpin/St"].value = 500  # in pN
+    fit["dna_ss_hairpin/St"].upper_bound = 2000  
+
+Fit all the data and plot the result::
+
+   >>> fit.fit()
+   >>> print(fit.params)
+   Name                  Value       Unit      Fitted   Lower bound    Upper bound
+   --------------      ------------ --------  -------  -------------  -------------
+   dna_handles/Lp        34.7309    [nm]        True    30              100
+   dna_handles/Lc         1.75753   [micron]    True     0.00034        inf
+   dna_handles/St       904.301     [pN]        True     1              inf
+   kT                     4.11      [pN*nm]     False    3.77           8
+   dna_ss_hairpin/Lp      0.785985  [nm]        True     0.5            2
+   dna_ss_hairpin/Lc      0.0215748 [micron]    True     0.001          inf
+   dna_ss_hairpin/St   2000         [pN]        True     1              2000
+
+As can be seen from the table, most fitted parameters converge and have values in the expected range. However, the stretch modulus of the hairpin hits the upper bound of 2000 pN, indicating that this parameter did not converge.
+When observing that a parameter does not converge, it is important to go back to the fit and see how it can be improved. In this case, increasing the upper bound for `dna_ss_hairpin/St` does not visually change the fit and does not result in convergence;
+the stretch modulus of the DNA handles and the hairpin cannot be optimized at the same time. A solution would be to use the freely jointed chain, instead of the extensible freely jointed chain to fit the unfolded hairpin, which is equivalent to setting
+a very large value for `dna_ss_hairpin/St`::
+
+    fit["dna_ss_hairpin/St"].upper_bound = 100000  # in pN
+    fit["dna_ss_hairpin/St"].value = 100000  
+    fit["dna_ss_hairpin/St"].fixed = True 
+    fit.fit()
+
+Plot the result::
+
+    plt.figure()
+    fit[dna_handles_force].plot()
+    fit[dna_handles_and_hairpin_force].plot()
+    plt.xlabel(r"Distance ($\mu$m)")
+    plt.ylabel("Force (pN)")
+    Lc_hairpin = fit["dna_ss_hairpin/Lc"].value*1000
+    plt.title(f"Fitted hairpin length is {Lc_hairpin:0.1f} nm")
+
+.. image:: fit_all.png
+
+The expected contour length for the hairpin was ~17 nm and the fitted length is 21.7 nm. 
+
+Check and improve fit quality
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The next step is to study the confidence intervals and quality of the fit, for example using the :ref:`likelihood profile <ple_confidence_intervals>`.
+The fit can be further improved by fitting multiple data sets at once, :ref:`Global fitting <global_fit>`.
+
+Unequal bead sizes
+^^^^^^^^^^^^^^^^^^
+
+In this example, the two bead sizes are equal. For unequal bead sizes, an extra distance offset can be :ref:`included in the model <model_composition>` to account for not perfectly centered templates.
```

Comment:
I really like that you added this section, but is the issue with template matching evident from just this?

```suggestion
In this example, the two bead sizes are equal. Since this is the case, we used a single template to track the bead positions. This means that even if a template is not centered perfectly, any offset negates, because both templates are offset in an identical manner.

When using different beads however, one uses two different templates, which means that the centering of the template becomes far more critical. As a result, we can end up with an offset in the bead-to-bead distance due to a slight off-centering of one or both of the templates.

When working with unequal bead sizes, an extra distance offset can be :ref:`included in the model <model_composition>` to account for not perfectly centered templates.
```

Reply:
yeah, this is explained in application guides about piezo tracking. But it does not hurt to repeat it here.